### ISSUE TITLE
it is not installing - PHP 7.4 / umbuntu server 18.4

### ISSUE THREAD

**Issue by murilolivorato** [pinepain/php-v8#115]
I am trying to install in PHP 7.4 / umbuntu server 18.4
I type those  commands - 
```
$ sudo add-apt-repository -y ppa:ondrej/php
$ sudo add-apt-repository -y ppa:pinepain/php
$ sudo apt-get update -y
$ sudo apt-get install -y php7.4 php-v8

```

Then
```
$ php --ri v8
show this error
Extension 'v8' not present.

```

please some one can help me ? I need this extension . thanks

**Comment by pinepain** [comment]
Hi @murilolivorato, what a bummer, I feel your frustration. Unfortunately, the packages in pinepain/php PPA were published on 2018-03-01, far before PHP 7.4 was released and were never updated after that as I step down from maintaining php-v8 ext (don't be confused by v8js ext, they are a bit different, v8js is still maintaned) and packaging around that time. The packaging scripts are available under https://github.com/phpv8/ppa-packaging repo, so you may build it yourself, the only change that might be required is to include newer ubuntu version(s) build.

**Comment by pinepain** [comment]
Closing as project is no longer actively maintained.
